### PR TITLE
Enable better integration with other tools

### PR DESCRIPTION
**Pull request by ferhatelmas**
Mainly for golangci-lint, which is the default tool.

Related to #2.

### CODE REVIEW

File: internal/errwrap/errwrap.go [hunk 1]
```diff
@@ -20,15 +20,8 @@ import (
...
-	Run:              run,
-	RunDespiteErrors: true,
-}
-
-func run(pass *analysis.Pass) (interface{}, error) {
+// Run is the runner for an analysis pass
+func Run(pass *analysis.Pass) (interface{}, error) {
 	inspect := pass.ResultOf[inspect.Analyzer].(*inspector.Inspector)
 
 	nodeFilter := []ast.Node{
```
Comment (fatih) [inline internal/errwrap/errwrap.go]: Just remove the `internal/` folder and move it to an upper level. So it should be `github.com/fatih/errwrap/errwrap`. And while you're at it, I think we can also remove the `cmd` folder. Move the `main.go` folder to the root so we can install it via `go get github.com/fatih/errwrap`.

Comment (ferhatelmas) [reply internal/errwrap/errwrap.go]: I didn't want to export everything but sounds cleaner. Updated.

**Comment by fatih** [comment]
Thank you for your contribution @ferhatelmas.

**Comment by ferhatelmas** [comment]
@fatih my pleasure!

**Pull request merged**